EpikCord/client/client.py

```diff
@@ -81,8 +81,8 @@ def stop_loop_on_completion(f: asyncio.Future):
     async def close(self):
         if not self.http.session.closed:
             await self.http.session.close()
-    
```

Comment:
![image](https://user-images.githubusercontent.com/53050011/215611129-49d091f4-cb9c-45dd-a88e-6e8e46091433.png)

Reply:
It was just a typo 👍